fw/http_tbl.c

```diff
@@ -156,6 +156,10 @@ tfw_http_tbl_scan(TfwMsg *msg, TfwHttpTable *table, TfwHttpActionResult *action)
 			return 0;
 		case TFW_HTTP_MATCH_ACT_BLOCK:
 			return -1;
+		case TFW_HTTP_MATCH_ACT_JSCH:
+			action->type = TFW_HTTP_RES_JSCH;
+			action->vhost = tfw_vhost_lookup_default();
```

Comment:
Its wrong. I add test to check it.
http_chain {{
            uri == "*1.html" -> jsch;
            host == "good" -> good;
            -> default;
        }}

For such configuration if host is good and uri == "*1.html" it should go to good vhost  not default!